x-pack/plugins/observability_solution/apm/server/routes/assistant_functions/get_observability_alert_details_context/get_context_message.ts

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { isEmpty } from 'lodash';
+
+export function getContextMessage({
```

Comment:
"Message" is quite generic. What about calling this something with "prompt" to make it clear that this is specifically meant for the LLM?

```suggestion
export function getApmAlertDetailsContextPrompt({
```